wire-ios-sync-engine/Source/Synchronization/Strategies/CallingRequestStrategy.swift

```diff
@@ -240,11 +240,6 @@ public final class CallingRequestStrategy: AbstractRequestStrategy, ZMSingleRequ
         events.forEach(processEvent)
     }
 
-    public func processEventsWhileInBackground(_ events: [ZMUpdateEvent]) {
```

Comment:
@netbe I think we should consider cherry-picking this fix to release

Reply:
@typfel @netbe I opened a PR to fix it on release. Please take a look: https://github.com/wireapp/wire-ios/pull/1569/files